frontend/modules/viewer/viewer.redux.ts

```diff
@@ -60,7 +60,8 @@ export const { Types: ViewerTypes, Creators: ViewerActions } = createActions({
 
 export const INITIAL_STATE = {
 	settings: window.localStorage.getItem('visualSettings') ?
-			JSON.parse(window.localStorage.getItem('visualSettings')) : DEFAULT_SETTINGS,
+		Object.assign(Object.assign({}, DEFAULT_SETTINGS), JSON.parse(window.localStorage.getItem('visualSettings')))
```

Comment:
This could make the whole line a little shorter/clearer
`settings: {...JSON.parse(window.localStorage.getItem('visualSettings')||'{}'), ...DEFAULT_SETTINGS}`

Reply:
fixed